docs/modules/configuring/con-virtualkafkacluster-tcp.adoc

```diff
@@ -0,0 +1,62 @@
+// file included in the following:
+//
+// kroxylicious-operator/assemblies/assembly-operator-deploy-a-proxy.adoc
+
+[id='con-configuring-virtualkafkacluster-{context}']
+= A virtual cluster for on-cluster access without TLS
+
+A `VirtualKafkaCluster` resource defines a logical Kafka cluster that is accessible to clients over the network.
+
+The virtual cluster references the following:
+
+* A `KafkaProxy` resource that the proxy is associated with.
+* One or more `KafkaProxyIngress` resources that expose the virtual cluster to Kafka clients.
+* A `KafkaService` resource that defined the backend Kafka cluster.
+* Zero or more `KafkaProtocolFilter` resources that apply filters to the Kafka protocol traffic passing between clients and the backend Kafka cluster.
+
+This example shows a `VirtualKafkaCluster`, exposing it to Kafka clients running on the same Kubernetes cluster.
+It uses plain TCP (as opposed to TLS) as the transport protocol.
+
+.Example `VirtualKafkaCluster` configuration.
+[source,yaml]
+----
+kind: VirtualKafkaCluster
+apiVersion: kroxylicious.io/v1alpha1
+metadata:
+  name: my-cluster
+  namespace: my-proxy
+spec:
+  proxyRef: # <1>
+    name: simple
+  targetKafkaServiceRef: # <2>
+    name: my-cluster
+  ingresses:
+    - ingressRef: # <3>
+        name: cluster-ip
+----
+<1> The `proxyRef` names the `KafkaProxy` resource that this virtual cluster is part of.
```

Comment:
```suggestion
<1> The `proxyRef` names the `KafkaProxy` associated with this virtual cluster.
```

Reply:
I think we should use whatever verb is the outcome of the [discussion above](https://github.com/kroxylicious/kroxylicious/pull/2107#discussion_r2072723428)

Reply:
Thanks for addressing teh comments @tombentley 
Host sounds good to me.
I've made a suggestion about the glossary items.

Reply:
```suggestion
<1> The `proxyRef` names the `KafkaProxy` hosting with this virtual cluster.
```